sdk/data/azappconfig/swagger_ext.json

```diff
@@ -0,0 +1,57 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Azure App Configuration"
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/{nextLink}": {
+      "get": {
+        "operationId": "GetNextPage",
+        "x-ms-pageable": {
+          "itemName": "items",
+          "nextLinkName": "@nextLink",
+          "operationName": "GetNextPage"
+        },
+        "parameters": [
+          {
+            "name": "nextLink",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+                "$ref": "https://github.com/Azure/azure-rest-api-specs/blob/e01d8afe9be7633ed36db014af16d47fec01f737/specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json#/definitions/KeyListResult"
```

Comment:
@timotheeguerin is there a way I can say "use appconfiguration.json that was included earlier" or do I have to specify a full path?  This same file is included in `autorest.md` above.